app/models/subnet/ipv6.rb

```diff
@@ -26,6 +26,10 @@ def self.supported_ipam_modes
     [:eui64, :db, :none]
   end
 
+  def self.supports_tftp?
+    false
```

Comment:
Why is tftp not supported? xinetd / tftpd which come with the foreman installer support IPv6